sync/class.jetpack-sync-queue.php

```diff
@@ -38,7 +38,16 @@ class Jetpack_Sync_Queue {
 	function __construct( $id ) {
 		$this->id           = str_replace( '-', '_', $id ); // necessary to ensure we don't have ID collisions in the SQL
 		$this->row_iterator = 0;
-		$this->random_int = random_int( 1, 1000000 );
```

Comment:
@zinigor Perhaps we should be using WordPress core fallback for `random_int`? which is in [this file](https://github.com/WordPress/WordPress/blob/master/wp-includes/random_compat/random_int.php).

It has to be [included like this](https://github.com/WordPress/WordPress/blob/master/wp-includes/compat.php#L433):
```php
if ( ! function_exists( 'random_int' ) ) {
	require ABSPATH . WPINC . '/random_compat/random.php';
}```